### ISSUE TITLE
Issue with "aria-controls": (short summary here please)

### ISSUE THREAD

**Issue by FunkyScript** [mdn/content#14757]
MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-controls

#### What information was incorrect, unhelpful, or incomplete?
'aria-control' should be 'aria-controls'

#### Specific section or headline?
First sentence within the Description section.

#### What did you expect to see?
aria-controls

#### Did you test this? If so, how?
Visually verified

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/accessibility/aria/attributes/aria-controls`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-controls
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/accessibility/aria/attributes/aria-controls/index.md
* Last commit: https://github.com/mdn/content/commit/614cd623fac41cbc68c54ef33b82dd4d5ebb04f7
* Document last modified: 2022-02-04T07:16:02.000Z

</details>

**Comment by sideshowbarker** [comment]
Thanks for raising this — I’ve opened #14766 with a fix.

After that fix is merged, this issue will automatically be marked as resolved, and closed.
